schema/Query.graphql

```diff
@@ -28,4 +28,14 @@ type Query {
     Generate challenge to authenticate with WebAuthn
     """
     generateAuthenticatorChallenge(username: String!): AuthenticationWithWebPublicKeyCredential
+
+    """
+    Retrieve a life information
+    """
+    getLife(id: String!): Life!
```

Comment:
`getLife` can return `null` if there's no match for the ID, so output should be `Life` instead of `Life!`

Reply:
also `id` should be of type `ObjectID`